robot/impl/robot_reconfigure_test.go

```diff
@@ -2638,46 +2664,28 @@ func TestRemoteRobotsGold(t *testing.T) {
 			sensors.Named(resource.DefaultServiceName),
 			datamanager.Named(resource.DefaultServiceName),
 			arm.Named("arm1"),
-			arm.Named("foo:pieceArm"),
-			audioinput.Named("foo:mic1"),
-			camera.Named("foo:cameraOver"),
-			movementsensor.Named("foo:movement_sensor1"),
-			movementsensor.Named("foo:movement_sensor2"),
-			gripper.Named("foo:pieceGripper"),
+			arm.Named("foo:remoteArm"),
```

Comment:
I recommend adding a comment to this `test.That` that we assert all of remote1's resources exist and none of remote2's.

Reply:
updated